media/css/security/security.less

```diff
@@ -57,6 +57,27 @@
         .font-size(32px);
         letter-spacing: normal;
     }
+    .cve {
+        margin: 2.5em 0 0 2.5em;
```

Comment:
Should this be `1em` on the left, to keep the content in line? `2.5em` feels a bit odd, especially at smaller screen sizes.